win32/devdoc/socket_transport_win32.md

```diff
@@ -0,0 +1,296 @@
+﻿# socket transport Win32
+
+## Overview
+
+socket_transport_win32 is the module that abstracts the socket transport for the windows platform
+
+The `SOCKET_TYPE` determines which API are applicable for this instance of socket_transport.  The following chart shows which API's are allowed under which type:
+
+| API                                    | Allowable Socket Type
```

Comment:
We should probably have a common doc in interfaces/devdoc (like the other modules do) for the common stuff like this table

Reply:
Added.